libvast/test/expression.cpp

```diff
@@ -254,9 +254,9 @@ TEST(validation - meta extractor) {
   expr = to<expression>("#field == \"id.orig_h\"");
   REQUIRE(expr);
   CHECK(caf::visit(validator{}, *expr));
-  expr = to<expression>("#field ~ \"orig\"");
+  expr = to<expression>("#field == \"orig\"");
   REQUIRE(expr);
-  CHECK(!caf::visit(validator{}, *expr));
+  CHECK(caf::visit(validator{}, *expr));
```

Comment:
This is a new test, the old one doesn't exist anymore because the match operator doesn't exist anymore. The diff just isn't very review-friendly here.